WordPress/WordPress.xcodeproj/project.pbxproj

```diff
@@ -8802,7 +8812,7 @@
 				SWIFT_OPTIMIZATION_LEVEL = "-Owholemodule";
 				SWIFT_VERSION = 3.0;
 				VALIDATE_PRODUCT = YES;
-				WPCOM_CONFIG = "$HOME/.wpcom_alpha_app_credentials";
+				WPCOM_CONFIG = $HOME/.wpcom_alpha_app_credentials;
```

Comment:
I don't think this changes should be here, can you double check why are the `"` being removed?